scripts/contentScript.js

```diff
@@ -380,6 +380,99 @@ async function FilesInPrBitbucket(response) {
 	});
 }
 
+const githubHunkHighlight = async (apiResponses) => {
+
+	// Todo : optomization needed to not highlight next deleted line space if not present in response.
+	let getFileName = Array.from(document.querySelectorAll('div[data-tagsearch-path]'));
+	getFileName.forEach(async (item) => {
+		let FileContent = item.getAttribute('data-tagsearch-path');
+		if (FileContent) {
+
+			let matchEncrypted = await sha256(FileContent);
+			let foundFile = null;
+			for (k in apiResponses["hunkinfo"]) {
+				let item = apiResponses["hunkinfo"][k];
+				if (item.file === matchEncrypted) {
+					foundFile = item;
+				}
+			}
+			if (foundFile) {
+				let value = Array.from(item.getElementsByTagName('tr'));
+				let changeBg = false;
+
+				value.forEach((items) => {
+					let secondRow = Array.from(items.getElementsByTagName('td'));
+					secondRow.forEach((item) => {
+						let buttonId = item.querySelector('button[data-line]');
+						if (buttonId) {
+							let dataLineValue = buttonId.getAttribute('data-line');
+							let tableContent = items.querySelector("td[data-split-side='left']");
+							if (tableContent) {
+								let checkDelete = tableContent.querySelector("span[data-code-marker='-']");
+								if (checkDelete) {
+									if (dataLineValue >= foundFile.line_start && dataLineValue <= foundFile.line_end) {
+										changeBg = true;
+										items.style.backgroundColor = 'rgb(86, 88, 0)';
+									}
+								}
+							}
+
+							if (tableContent) {
+								if (tableContent.innerHTML === '') {
+									if (changeBg) {
+										items.style.backgroundColor = 'rgb(86, 88, 0)';
+									}
+
+								}
+							}
+						}
+					})
+				})
+			}
+		}
+	})
+}
+
+const bitBucketHunkHighlight = (apiResponses) => {
+
+	const articles = document.querySelectorAll('article[aria-label^="Diff of file"]');
+	articles.forEach(async (article) => {
+		const ariaLabel = article.getAttribute('aria-label');
+		const fileName = ariaLabel.substring(13);
```

Comment:
Low LOGAF:
Why do we need to hardcode this number? Is this the best way to handle this?

```suggestion
		const fileName = ariaLabel.substring("Diff of file".length);
```